example/docker-compose.yaml

```diff
@@ -0,0 +1,55 @@
+version: '3.7'
+
+volumes:
+    prometheus_data: {}
+    grafana_data: {}
+
+networks:
+  prometheus_network:
+
+services:
+  prometheus_speedtest:
+    image: jraviles/prometheus_speedtest:latest
+    ports:
+      - 9516:9516
+    networks:
+      - prometheus_network
+    restart: always
+
+  prometheus:
+    image: prom/prometheus:latest
+    volumes:
+      - ./:/etc/prometheus/
+      - prometheus_data:/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
+      - '--web.console.templates=/usr/share/prometheus/consoles'
+    ports:
+      - 9090:9090
+    links:
+      - prometheus_speedtest:prometheus_speedtest
+    depends_on:
+      - prometheus_speedtest
+    networks:
+      - prometheus_network
+    restart: always
+
+  grafana:
+    image: grafana/grafana:latest
+    user: "472"
+    depends_on:
+      - prometheus
+    ports:
+      - 3000:3000
+    volumes:
+      - grafana_data:/var/lib/grafana
+      - ./grafana/provisioning/:/etc/grafana/provisioning/
```

Comment:
Is there anything interesting in this provisioning directory? Can we store it in `grafana_data` instead?